lib/frontend/src/frontend.rs

```diff
@@ -605,12 +605,12 @@ impl<'a> FunctionBuilder<'a> {
             return;
         }
 
+        let int_type = Type::int((access_size * 8) as u16).unwrap();
```

Comment:
Light suggestion: should we rename `size` and `access_size` to `byte_size` and `access_byte_size` to make this clearer? (maybe for callers of `emit_small_memcpy` too)

Reply:
I'm sympathetic, but my sense is it's not super important in this context, because the memcpy/memset/etc. interfaces are pretty well known.